### PR TITLE
Blender mapping

### PR DESCRIPTION
**Pull request by un-tone**
In scope of https://github.com/Starcounter/CompanyTrack/issues/292

### CODE REVIEW

File: src/UserAdmin/Api/LauncherHooks.cs [hunk 1]
```diff
@@ -93,11 +93,11 @@ public static void Register()
...
-            UriMapping.Map("/useradmin/menu", UriMapping.MappingUriPrefix + "/menu");
-            UriMapping.Map("/useradmin/app-name", UriMapping.MappingUriPrefix + "/app-name");
-            UriMapping.Map("/useradmin/search/{?}", UriMapping.MappingUriPrefix + "/search?query={?}");
-            UriMapping.Map("/useradmin/partial/unauthenticated?return_uri={?}", UriMapping.MappingUriPrefix + "/userform?{?}");
-            UriMapping.Map("/useradmin/user/authentication/settings/{?}", UriMapping.MappingUriPrefix + "/systemuser/authentication/settings/{?}");
+            Blender.MapUri("/useradmin/menu", "/menu");
+            Blender.MapUri("/useradmin/app-name", "/app-name");
+            Blender.MapUri("/useradmin/search/{?}", "/search?query={?}");
+            Blender.MapUri("/useradmin/partial/unauthenticated?return_uri={?}", "/userform?{?}");
+            Blender.MapUri("/useradmin/user/authentication/settings/{?}", "/systemuser/authentication/settings/{?}");
```
Comment (warpech) [inline src/UserAdmin/Api/LauncherHooks.cs]: @alemoi pls help me to review this... The above is technically correct, but in your mind did you imagine string tokens that look like URIs? Maybe it is better to have tokens that do not begin with `/`, to emphasize that they are not URIs?

Comment (alemoi) [inline src/UserAdmin/Api/LauncherHooks.cs]: Yes, I think tokens should be thought of class names, so not URIs.

Comment (warpech) [inline src/UserAdmin/Api/LauncherHooks.cs]: I agree. 

@un-tone for the beginning, could you just remove `/` from the beginning of all map tokens?

Comment (warpech) [inline src/UserAdmin/Api/LauncherHooks.cs]: @alemoi  So should this work?

```cs
Blender.MapUri("/useradmin/search/{?}", "search");
Blender.MapUri("/useradmin/partial/unauthenticated?return_uri={?}", "userform");
Blender.MapUri("/useradmin/user/authentication/settings/{?}", "systemuser/authentication/settings");
```

Comment (alemoi) [inline src/UserAdmin/Api/LauncherHooks.cs]: > systemuser/authentication/settings

This is quite long but it works, just to decrease the number of work :)

Comment (un-tone) [inline src/UserAdmin/Api/LauncherHooks.cs]: Fixed all tokens.

**Comment by warpech** [comment]
https://github.com/Starcounter/CompanyTrack/issues/292 is done and released in Starcounter 2.3, so this can be merged now. Thanks @un-tone for the great work on this!

**Pull request merged**